1-setup-windows-msys2.sh

```diff
@@ -28,25 +28,26 @@ libtool \
 make \
 patch \
 tar \
-$MINGW_PACKAGE_PREFIX-gcc \
+$MINGW_PACKAGE_PREFIX-boost \
+$MINGW_PACKAGE_PREFIX-cairomm \
 $MINGW_PACKAGE_PREFIX-ccache \
 $MINGW_PACKAGE_PREFIX-cmake \
-$MINGW_PACKAGE_PREFIX-libltdl \
-$MINGW_PACKAGE_PREFIX-make \
-$MINGW_PACKAGE_PREFIX-pkgconf \
 $MINGW_PACKAGE_PREFIX-dlfcn \
-$MINGW_PACKAGE_PREFIX-SDL2  \
-$MINGW_PACKAGE_PREFIX-boost \
-$MINGW_PACKAGE_PREFIX-cairomm \
 $MINGW_PACKAGE_PREFIX-ffmpeg \
 $MINGW_PACKAGE_PREFIX-fftw \
+$MINGW_PACKAGE_PREFIX-gcc \
 $MINGW_PACKAGE_PREFIX-glibmm \
+$MINGW_PACKAGE_PREFIX-gtkmm3 \
 $MINGW_PACKAGE_PREFIX-imagemagick \
+$MINGW_PACKAGE_PREFIX-libltdl \
+$MINGW_PACKAGE_PREFIX-libmng \
 $MINGW_PACKAGE_PREFIX-libxml++2.6 \
-$MINGW_PACKAGE_PREFIX-pango \
-$MINGW_PACKAGE_PREFIX-gtkmm3 \
+$MINGW_PACKAGE_PREFIX-make \
+$MINGW_PACKAGE_PREFIX-ninja \
 $MINGW_PACKAGE_PREFIX-openexr \
-$MINGW_PACKAGE_PREFIX-libmng
+$MINGW_PACKAGE_PREFIX-pango \
+$MINGW_PACKAGE_PREFIX-pkgconf \
+$MINGW_PACKAGE_PREFIX-SDL2
```

Comment:
As far as I know, the only reason we depend on SDL2 is `melt` of `mlt`.
Since we don't use `'melt`, is there a way to avoid building it to avoid SDL2 dependency?

Reply:
I will check it.

Reply:
It may be another PR :)

Reply:
> It may be another PR :)

Yes. I just thought about the HarfBuzz dependency. Should it be added?

Reply:
If we trust on MSYS2 as our "package provider", it makes `gtkmm3` depends on `pangomm` that depends on `pango` that (finally) depends on `harfbuzz`! :)
But I'd make sure `harfbuzz` is listed there anyway. Because `synfig-core` uses `harfbuzz` anyway.